k8s/namespaces/money-claims/cmc-claim-store/aat.yaml

```diff
@@ -4,19 +4,12 @@ metadata:
   name: cmc-claim-store
   namespace: money-claims
 spec:
-  releaseName: cmc-claim-store
-  rollback:
-    enable: true
-    retry: true
   chart:
     git: git@github.com:hmcts/hmcts-charts
     ref: master
```

Comment:
Already referenced in base chart

Reply:
Chart section removed from base chart due to different environments using different sources